homeassistant/components/smappee_official/strings.json

```diff
@@ -0,0 +1,13 @@
+{
+    "config": {
+        "step": {
+            "pick_implementation": {
+                "title": "Pick Authentication Method"
+            }
+        },
+        "abort": {
+            "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]",
+            "missing_configuration": "The component is not configured. Please follow the documentation."
+        }
```

Comment:
We need to add the `"authorize_url_timeout"` string back too, that were present in the scaffold template.